src/api.ts

```diff
@@ -62,7 +57,10 @@ export const extractCommitFromImage = (imageName: string): CommitHash => imageNa
  */
 export async function refreshLocalImages() {
 	const images = await docker.listImages();
-	state.localImages = new Map( images.map( 
+	const isTag = (tag: string) => tag.startsWith( config.build.tagPrefix );
+	const hasTag = (image: Docker.ImageInfo) => image.RepoTags.some( isTag );
+
+	state.localImages = new Map( images.filter( hasTag ).map( 
 		image => [ image.RepoTags[0], image ] as [ string, Docker.ImageInfo ]
```

Comment:
@samouri and @sirreal I'm hoping that this line is causing the images to not finish building - just taking the first tag instead of one with the prefix in it. I have updated to use `find( isTag )` instead of grabbing the first one